Doc/c-api/long.rst

```diff
@@ -337,6 +369,52 @@ distinguished from a number.  Use :c:func:`PyErr_Occurred` to disambiguate.
       This function will no longer use :meth:`~object.__int__`.
 
 
+.. c:function:: int PyLong_ToInt32(PyObject *obj, int32_t *value)
+
+   Return a signed C :c:expr:`int32_t` representation of *obj*.
+
+   If the *obj* value is out of range, raise an :exc:`OverflowError`.
+
+   Set *\*value* and return ``0`` on success.
+   Set an exception and return ``-1`` on error.
+
+   .. versionadded:: 3.14
+
+
+.. c:function:: int PyLong_ToInt64(PyObject *obj, int64_t *value)
+
+   Similar to :c:func:`PyLong_ToInt32`, but return a signed C
+   :c:expr:`int64_t` representation.
```

Comment:
Can we instead group the definitions together? I'm sure I saw someone say this syntax should work:

```
.. c:function::
    int PyLong_ToInt32(PyObject *obj, int32_t *value)
    int PyLong_ToInt64(PyObject *obj, int64_t *value)

    Docs for both
```